### ISSUE TITLE
use Language Model to decode other NN's output

### ISSUE THREAD

**Issue by bjtommychen** [mozilla/DeepSpeech#1597]
Hi, all
        My NN model is a modified DeepSpeech. whose WER is 0.15 without LM.
        So, I want to use this Language Model to get better WER.
        My step:
        1. run NN to get my NN's output (logits).
        2. modify native_client/client.cc
               a) push_back the logits into StreamingState->accumulated_logits.
               b)DS_IntermediateDecode()
        In this way, I can get it work. but WER only 0.11, I hope it should be < 0.09.

        I did below, is there something more I missed ? pls advice. Thanks.
        1. adjust beam-width/lm_weight/wordcnt_weight to get the best result.
        2. what's the input of Language Model. FC's output, or log() of it? thanks.

**Comment by lissyx** [comment]
@bjtommychen Thanks for sharing this, it obviously looks interesting, but this is not a bug in our codebase per-se, so please use Discourse for that kind of discussion :).
https://discourse.mozilla.org/c/deep-speech
